docs/deployment/production-deployment.rst

```diff
@@ -144,13 +144,14 @@ The last thing to do is to update our ``application.yml`` file. We need to add `
 
     # repository settings (can be changed to different repository)
     repository:
-        type: 5
-        blazegraph:
-            url: http://blazegraph:8080/blazegraph
+        type: 4
```

Comment:
I miss the part of username and pwd

Reply:
Added in [984bbd2](https://github.com/FAIRDataTeam/FAIRDataPoint-Docs/pull/56/commits/984bbd25ca8a62a874e8fc4fc3cb71d9a6b135f3)